.lgtm/cpp-queries/bool-type-mismatch-return.qhelp

```diff
@@ -30,7 +30,7 @@ A boolean macro - <code>true</code> or <code>false</code>
 A comparison - <code>==</code> or <code>!=</code> (or less than / greater than variants).
 </li>
 <li>
-A logical operation - <code>&&</code> or <code>||</code>
+A logical operation - AND or OR.
```

Comment:
You could use `&amp;`.

Reply:
I'm paranoid at this point, will add it back later.